src/clr-angular/utils/drag-and-drop/_drag-and-drop.clarity.scss

```diff
@@ -0,0 +1,24 @@
+@include exports("drag-and-drop.clarity") {
+    .drag-handle {
+        cursor: grab;
+    }
+    .draggable-ghost {
+        position: fixed;
+        visibility: hidden;
+        z-index: 2147483647; // max z-index;
+        pointer-events: none;
+        will-change: left, top;
+        opacity: 0.6;
+        transform: translate3d(0px, 0px, 0);
```

Comment:
Why the `transform`? As far as I see, this is unused since you are animating `top` and `left`.

Reply:
Yes, I can remove it now. It doesn't offer any benefits here.

Reply:
Removed `transform`.